h/views/api/errors.py

```diff
@@ -36,6 +37,13 @@ def api_notfound(request):
     return {"status": "failure", "reason": message}
 
 
+@json_view(context=OAuthAuthorizeError, renderer="h:templates/oauth/error.html.jinja2")
```

Comment:
This was using `@view_config` before but is using `@json_view` now which implies JSON input/output. Is that intentional?

Reply:
Oops, no that's a mistake, thanks

Reply:
Fixed